hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/balancer/TestBalancer.java

```diff
@@ -2309,8 +2310,11 @@ public void testBalancerWithSortTopNodes() throws Exception {
     // Hence, overall total blocks moved by HDFS balancer would be either of these 2 options:
     // a) 2 blocks of total size (100B + 100B)
     // b) 3 blocks of total size (50B + 100B + 100B)
-    assertTrue(balancerResult.getBytesAlreadyMoved() == 200
-        || balancerResult.getBytesAlreadyMoved() == 250);
+    assertTrue(
```

Comment:
might be helpful if we add `balancerRestult` into the error message as the assertion is getting longer; so it will be easier to reason about in case of test failure. At least it's easier than checking logs in console.

Reply:
Sure, on it.